i18n/locales/en.ts

```diff
@@ -217,7 +217,7 @@ const translations = {
       title: 'Outro',
       nav_title: 'Off to the address',
       paragraph_one:
-        'You run down to your garage, step into your dad’s old solocopter, punch in the coordinates, and make your way to the address.',
+        'You run down to your garage, step into your dad’s old Budgetcopter, punch in the coordinates, and make your way to the address.',
```

Comment:
At some point between chapters 1 and 2 solocopter was changed to Budgetcopter . That might be intentional (maybe a Budgetcopter is a type of solocopter?) but I updated the copy and replaced "solocopter" with "Budgetcopter" just to keep things simple.